### PR TITLE
Feature/plausible analytics new

### PR DESCRIPTION
**Pull request by julia-rom**
Created a new PR as per Anthony's comment for the previous one:

> the issue w/ the plausible branch is the buildpack migration has been merged into the plausible branch but buildpacks are only automatically provisioned on app creation. Options are to close/open a new PR so that a new review app is created, or manually replace nodejs with the create-react-app buildpack and redeploy

[new review app link - place ID](https://dtpr-sticker-feature-pl-k9ipae.herokuapp.com/places/Ur-FM0fJw)

[new review app link - sensor ID](https://dtpr-sticker-feature-pl-k9ipae.herokuapp.com/sensors/-MSTrI2eY2KgpnGr4E4f)

### CODE REVIEW

**Comment by tonybennett** [comment]
Per last PR, code looks good, just need @turnKei to validate everything is coming through correctly now that the url is working

File: public/index.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="%REACT_APP_GOOGLE_FONTS_LINK%" rel="stylesheet" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!-- fathom analytics -->
-    <script src="https://cdn.usefathom.com/script.js" data-site="%REACT_APP_FATHOM_DATA_SITE%" defer></script>
+    <!-- analytics -->
+    <script defer data-domain="%REACT_APP_DOMAIN_URL%,%REACT_APP_AGGREGATED_DOMAIN_URL%"src="https://plausible.io/js/plausible.local.js"></script>
```
Comment (turnKei) [inline public/index.html]: This is still pointing to local, so won't work when deployed. As discussed this should be changed to `https://plausible.io/js/plausible.js`

**Review by turnKei — CHANGES_REQUESTED** [review]
still pointing to local env, should be the standard per plausible docs

**Comment by turnKei** [comment]
Not showing up in Plausible, looks like you forgot to add those config vars to the review app?

**Review by turnKei — APPROVED** [review]

**Pull request merged**